octopus_deploy/tests/test_unit.py

```diff
@@ -13,7 +13,7 @@
 from datadog_checks.dev.utils import get_metadata_metrics
 from datadog_checks.octopus_deploy import OctopusDeployCheck
 
-from .constants import ALL_METRICS
+from .constants import ALL_DEPLOYMENT_LOGS, ALL_METRICS, ONLY_TEST_LOGS
 
 MOCKED_TIME1 = datetime.datetime.fromisoformat("2024-09-23T14:45:00.123+00:00")
 MOCKED_TIME2 = MOCKED_TIME1 + datetime.timedelta(seconds=15)
```

Comment:
```suggestion
MOCKED_TIME2 = MOCKED_TIME1 + datetime.timedelta(seconds=DEFAULT_COLLECTION_INTERVAL)
```

Could we remove the `15` and replace it with some constant? If my understanding is correct, I guess it's the default collection interval.

Reply:
good idea! I moved these to `constants.py` too